<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>431</width>
      <height>250</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>215</number>
    </attribute>
    <column>
     <property name="text">
      <string>Pokémon</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipos</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="Generacion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokédex Generacion </string>
    </property>
   </widget>
   <widget class="QLabel" name="numGen">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="regresar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="datos">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../EXTRAS/fondo.jpg</pixmap>
    </property>
   </widget>
   <zorder>img</zorder>
   <zorder>tabla</zorder>
   <zorder>Generacion</zorder>
   <zorder>numGen</zorder>
   <zorder>regresar</zorder>
   <zorder>datos</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
